frontend/www/js/omegaup/components/problem/CollectionProblem.vue

```diff
@@ -81,7 +80,7 @@ export default class CollectionProblem extends Vue {
   }
 
   getProblemLevelIcon(problemLevel: string): string {
-    if (problemLevelIcons.hasOwnProperty(problemLevel))
+    if ({}.hasOwnProperty.call(problemLevelIcons, problemLevel))
```

Comment:
```suggestion
    if (Object.prototype.hasOwnProperty.call(problemLevelIcons, problemLevel))
```

este es el patrón preferido. así no se tiene que construir un objeto (el `{}`) nada más para invocar el método.